Assets/Scripts/Singletons/DataModelSingletons/PilotsManager.cs

```diff
@@ -69,12 +69,12 @@ public void HirePilot(Pilot pilot)
 
     public Pilot[] GetHiredPilots()
     {
-        return Instance.Pilots.Where(p => p.IsHired) as Pilot[];
+        return Instance.Pilots.Where(p => p.IsHired).ToArray();
     }
 
     public Pilot[] GetPilotsForHire()
     {
-        return Instance.Pilots.Where(p => !p.IsHired) as Pilot[];
+        return Instance.Pilots.Where(p => !p.IsHired).ToArray();
```

Comment:
cant believe i didn't catch this the first time around lmao

Reply:
i dont remember doing this lol
i think i had it as toarray originally as it was working, then did this for some reason.